ruby-challenges.rb

```diff
@@ -10,6 +10,26 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+# Input: and array and a single letter 
+# Output:the words with the single letter includded
+# Psuedocode: so i want to create a function the can take in 2 pareameters. One parameter is going to be the array of words, and the second parameter is going to be the single letter . the way it will work is by checking the value in each index of the array and using the include? method to see if the index has that letter in it . im thinking about doing an if statment to make this happen 
+
+def the_searcher1 array
+array.select do |value|
+    value.include? 't'
```

Comment:
This is a great starting point! Just like with functions, we can have multiple parameters for methods.